play/src/components/joingame/EnterGameCode.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { styled, useTheme } from '@mui/material/styles';
+import { Stack, Box, Typography } from '@mui/material';
+import {
+  IntroButton,
+  IntroTextField,
+  JoinGameBackgroundContainer,
+} from '../../lib/styledcomponents/StyledComponents';
+import Logo from '../../img/rightOnLogo.svg';
+
+const StackContainer = styled(Stack)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  maxWidth: theme.breakpoints.values.xs,
+}));
+
+const PaddedContainer = styled(Box)(({ theme }) => ({
+  paddingLeft: `${theme.sizing.smallPadding}px`,
+  paddingRight: `${theme.sizing.smallPadding}px`,
+}));
+
+interface EnterGameCodeProps {
+  gameCodeValue: string;
+  handleGameCodeChange: (newValue: string) => void;
+  inputError: boolean;
+}
+
+export default function EnterGameCode({
+  gameCodeValue,
+  handleGameCodeChange,
+  inputError,
+}: EnterGameCodeProps) {
+  const theme = useTheme();
+  return (
+    <JoinGameBackgroundContainer>
+      <StackContainer spacing={5}>
+        <img
+          style={{
+            width: '214px',
+            height: '118px',
+            paddingTop: `${theme.sizing.extraLargePadding}px`,
+          }}
+          src={Logo}
+          alt="Question"
+        />
+        <Box>
+          {/* container here to trim the spacing set by parent stack between text and input, typ */}
+          <Typography variant="h2" sx={{ weight: 700, textAlign: 'center' }}>
+            Enter Game Code
+          </Typography>
+          <IntroTextField
+            fullWidth
+            variant="filled"
+            autoComplete="off"
+            onChange={(newValue) => {
+              handleGameCodeChange(newValue.target.value);
+            }}
+            onFocus={(newValue) => {
+              if (newValue.target.value === '####') {
+                handleGameCodeChange('');
+              }
```

Comment:
Can you use a placeholder prop instead?

Reply:
Way better. Fixed here and on player name input fields in HowToPlay PR